errors.go

```diff
@@ -155,6 +155,9 @@ type FieldError interface {
 	// NOTE: is not registered translation can be found it returns the same
 	// as calling fe.Error()
 	Translate(ut ut.Translator) string
+
+	// Error returns the FieldError's error message
+	Error() string
```

Comment:
Hey @joshhsoj1902 thanks for the PR

I think that only this addition is necessary

Reply:
Hey @joeybloggs, I might be doing something wrong on my end, But for me, it only works when I also change the `*fieldError` references to the interface. When I just add `Error() string` to the interface I still get this error thrown

>panic: interface conversion: validator.FieldError is *validator.fakeFieldError, not *validator.fieldError 

(that panic happens on line 51, `fakeFieldError` is my implentation of `FieldError`)

My understanding is that since line 51 is trying to cast to the private struct, the only way the casting will pass is if the object being cast was also made from that private struct. and since it's private, I have no way of actually doing that.

Reply:
I'll take a closer look, sorry for the delay I'm extremely busy

Reply:
No problem at all. If there is anything you would rather I do instead, let me know. I think there are a few ways to solve this.

Reply:
@joshhsoj1902 would you be able to share your example, seems like the best place for me to start :)

Reply:
Hey Joey, sorry for the slow reply. I do plan on circling back and creating a small sample program for you. Time is always the issue. Things should slow down a bit for me after next week.

Reply:
no worries I only get a couple hours at most to take care of my open source stuff lately, tis the holiday season.